di-ipv-core-stub/Dockerfile

```diff
@@ -21,7 +21,6 @@ COPY --from=build \
 	/app/
 
 WORKDIR /app
-ADD config config
```

Comment:
Why are you removing the adding of the config directory?

Reply:
I'm not that fussed about whether the image is public or private - I just need it somewhere that it can be easily accessed. The config on the other hand is definitely private, and I don't want an entire config repo exposed for a few settings.

So the approach is build the image so it works without config. Locally it can be run with a sibling path to the config. In a PR we get the necessary config from a S3 bucket, which reduces the blast radius if something went wrong.